roon.json.in

```diff
@@ -32,5 +32,6 @@
   \"dependencies\" : [
       { \"name\" : \"app\", \"version\" : \"0.2.0\" },
       { \"name\" : \"remoteOS\", \"version\" : \"0.2.0\" }
-  ]
+  ],
```

Comment:
Could you please add the mdns address here? I don't know if this is correct, but I assume something similar:
`  \"mdns\": \"_roon._tcp\",`

Reply:
I believe Roon  doesn't used mdns for discovery but let me  check.

Reply:
Nope, no mdns for Roon.

Reply:
https://github.com/RoonLabs/node-roon-api/blob/master/sood.js

Reply:
Looks like we need to implement other discovery methods :)